stmhal/boards/common.ld

```diff
@@ -15,7 +15,11 @@ SECTIONS
 
         . = ALIGN(4);
         */ff.o(.text*)
-        */stm32f4xx_hal_sd.o(.text*)
+	*/vfs_fat_*.o(.text*)
+
+	*/py/formatfloat.o(.text*)
+	*/py/parsenum.o(.text*)
+	*/py/mpprint.o(.text*)
```

Comment:
In the makefile there are rules to always use -Os for these files, which probably needs updating.